content/events/release-apache-skywalking-python-1-0-0/index.md

```diff
@@ -12,6 +12,7 @@ SkyWalking Python 1.0.0 is released! Go to [downloads](/downloads) page to find
 **DockerHub Image**: https://hub.docker.com/r/apache/skywalking-python
 
 - **Important Notes and Breaking Changes:**
+  - The new PVM metrics reported from Python agent requires SkyWalking OAP v9.3.0 to show out-of-the-box.
```

Comment:
If python agent is somehow version sensitive for v9, please document here.

https://github.com/apache/skywalking/blob/master/docs/en/setup/service-agent/agent-compatibility.md

Reply:
Sure, I will add it later!

Reply:
Could you be clear, about why does python need 9.3.0? Which protocol is new in 9.3.0?

Reply:
If it's only the PVM metrics visualization, then I'd consider it "compatible" because it just works, it doesn't required users to update OAP just in order to upgrade to 1.0.0. @Superskyyy please confirm whether there is anything else that breaks.

Reply:
PVM is fine. Meter system is there for a long time. The only thing is, user may need MAL script of PVM from a newer version of OAP. But generally, compatibility is still there, nothing wrong would happen w/o that script file.

Reply:
So, if you want to highlight this, try to write how to activate PVM in older OAP version, or just don't say anything until there is anyone asked.

Reply:
I don't think there's other compatibility issues, it's just the PVM tab wouldn't be there out-of-the-box, and no real broken things would happen as said - just need a mal script. 

> So, if you want to highlight this, try to write how to activate PVM in older OAP version, or just don't say anything until there is anyone asked.

Got it!

Reply:
I think I won't add it right now, let's wait and see if anyone asks.